packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx

```diff
@@ -3,20 +3,77 @@ import { Meta } from "@storybook/addon-docs";
 <Meta title="Docs/Framework Integration/Vue" />
 
 # Integrate with Vue.js (as Web Components)
-
-> We set up a demo project with Vue.js to show you how to integrate the components into a newly generated Vue.js
+> We set up a demo project with Vue.js (v3) to show you how to integrate the components into a newly generated Vue.js
 > project. Scroll down to see the Stackblitz.
 >
 > These instructions are based on `vue@2.6.11` and [`create-react-app@4.2.3`](https://cli.vuejs.org/).
 
-## 0) Prepare your Project
+## Vue.js 3
 
-Before using `@inovex.de/elements`, you should make sure that your current setup works and boots properly.
+Please follow these steps for integration into Vue.js(>= 3.0.0):
+
+### 0) Prepare your Project
+Before using `@inovex.de/elements-vue`, you should make sure that your current setup works and boots properly.
 If this is the case, proceed to the next step.
 
-## 1) Install `@inovex.de/elements`
+### 1) Install `@inovex.de/elements-vue`
+Add the package `@inovex.de/elements-vue` to your project using **npm**:
+
+```sh
+$ npm install @inovex.de/elements-vue --save
+```
+
+or **yarn**:
+
+```sh
+$ yarn add @inovex.de/elements-vue
+```
+
+### 2) Prepare the Project to use vue wrapper
+Before you start using the elements icons, you need to prepare the project.
+Open `src/main.js` and add the following code snippet on the first lines to use the elements vue wrapper.
+
+```js
+// src/main.js
+import { createApp } from 'vue';
+import { InoElementsVue } from '@inovex.de/elements-vue';
+import App from './App.vue'
+
+const app = createApp(App);
+app.use(InoElementsVue);
```

Comment:
Maybe we should provide a comment to this line to make it clear that this is the line the consumer has to add

e.g.: `app.use(InoElementsVue); // add this line to your code`